service-admin/src/App/src/ConfigProvider.php

```diff
@@ -63,6 +63,7 @@ public function getDependencies() : array
                 Middleware\Authorization\AuthorizationMiddleware::class => Middleware\Authorization\AuthorizationMiddlewareFactory::class,
                 Middleware\Session\SessionMiddleware::class             => Middleware\Session\SessionMiddlewareFactory::class,
                 Middleware\ViewData\ViewDataMiddleware::class           => Middleware\ViewData\ViewDataMiddlewareFactory::class,
+                Middleware\UserValidation\UserValidationMiddleware::class           => Middleware\UserValidation\UserValidationMiddlewareFactory::class,
```

Comment:
This should be reindented so all the '=>' line up as they do in the lines above.